### PR TITLE
Fix wrong file size "find" command

### PR DESCRIPTION
**Pull request by noculture**
The command to find files above 100MB in size was wrong. It found files above 50MB instead.

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/linux/user-and-file-management/file-management/using-find-to-search-by-file-size.md?ref=b48e7eff97aa331b2befad8f2f864efaccd1ac15

File: linux/user-and-file-management/file-management/using-find-to-search-by-file-size.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ The `find` command is used to search for files on your computer.  It's often use
 This will find all files at least 50MB or larger in the current directory:
```
Comment (nem035) [inline linux/user-and-file-management/file-management/using-find-to-search-by-file-size.md]: We should also change the `50MB` to say `100MB` in order to match the command bellow (and the rest of the text underneath it).

**Comment by nem035** [comment]
Thanks for the contribution @noculture ❤️

File: linux/user-and-file-management/file-management/using-find-to-search-by-file-size.md (hunk 1 above)
Comment (noculture) [reply linux/user-and-file-management/file-management/using-find-to-search-by-file-size.md]: Thanks for spotting that!

**Comment by nem035** [comment]
Your changes will be live in the app shortly 😄

**Pull request merged**